mev_inspect/compound_liquidations.py

```diff
@@ -0,0 +1,126 @@
+    decimals = token_instance.functions.decimals().call()
+    return decimals
+
+def get_compound_liquidations(
+    traces: List[ClassifiedTrace],
+    comp_markets: Dict[str,str],
+    w3: Web3,
+) -> List[Liquidation]:
+
+    """Inspect list of classified traces and identify liquidation"""
+    liquidations: List[Liquidation] = []
+
+    for trace in traces:
+
+        if (
+            trace.classification == Classification.liquidate
+            and trace.protocol == Protocol.compound_v2
+        ):
+            # First, we look for cEther liquidations (position paid back via tx.value)
+            child_traces = get_child_traces(trace.transaction_hash, trace.trace_address, traces)
+            seize_trace = _get_seize_call(child_traces)
+            c_token_collateral = trace.inputs["cTokenCollateral"]
+            underlying_exchange_rate = get_underlying_ctoken_exchange_rate(c_token_collateral, trace.block_number, w3, comp_markets)
+            if trace.abi_name =='CEther':
+                liquidations.append(
+                    Liquidation(
+                        liquidated_user=trace.inputs["borrower"],
+                        collateral_token_address=WETH_ADDRESS, # WETH since all cEther liquidations provide Ether
+                        debt_token_address=comp_markets[c_token_collateral],
+                        liquidator_user=seize_trace.inputs["liquidator"],
+                        debt_purchase_amount=trace.value,
+                        protocol=Protocol.compound_v2,
+                        received_amount=seize_trace.inputs["seizeTokens"] * underlying_exchange_rate,
+                        transaction_hash=trace.transaction_hash,
+                        trace_address=trace.trace_address,
+                        block_number=trace.block_number,
+                    )
+                )
+            elif trace.abi_name == 'CToken': # cToken liquidations where liquidator pays back via token transfer
+                liquidations.append(
+                    Liquidation(
+                        liquidated_user=trace.inputs["borrower"],
+                        collateral_token_address=comp_markets[trace.to_address], # Find out what collateral is being provided by looking at to address (cToken)
+                        debt_token_address=comp_markets[c_token_collateral],
+                        liquidator_user=seize_trace.inputs["liquidator"],
+                        debt_purchase_amount=trace.inputs["repayAmount"],
+                        protocol=Protocol.compound_v2,
+                        received_amount=seize_trace.inputs["seizeTokens"] * underlying_exchange_rate,
+                        transaction_hash=trace.transaction_hash,
+                        trace_address=trace.trace_address,
+                        block_number=trace.block_number,
+                    )
+                )
+    return liquidations
+
+
+def _get_seize_call(child_traces: List[ClassifiedTrace]) -> Optional[ClassifiedTrace]:
```

Comment:
small but better to call this `traces` if the fact that they're children doesn't matter for the job the function does